src/test/scala/chiselTests/BundleLiteralSpec.scala

```diff
@@ -47,6 +46,24 @@ class BundleLiteralSpec extends ChiselFlatSpec with Utils {
     } }
   }
 
+  "bundle literals" should "not pack when sparsely specified" in {
```

Comment:
Should this have gone in its own PR? Doesn't look related to Vec literals?

Reply:
Removed